style.css

```diff
@@ -350,6 +762,114 @@ body {
   .range__link {
     font-size: 12px;
   }
+
+  .order {
+    padding: 8vw 5vw 0 6.5vw;
+  }
+
+  .order-images {
+    margin-right: 7vw;
+  }
+
+  .order-image__first {
+    margin-top: 110px;
+    height: 55%;
+    width: 70%;
+  }
+
+  .order-image__second {
+    height: 90%;
+  	width: 95%;
+  	margin: 0;
+  }
+
+  .order-details {
+    padding-top: 11vw;
+  }
+
+  .order-content {
+    margin-bottom: 5vw;
+    padding-left: 5.5vw;
+    padding-right: 5.5vw;
+    padding-top: 6vw;
+    width: 550px;
+  }
+
+  .order__title {
+    font-size: 36px;
+    line-height: 38px;
+  }
+
+  .order__subtitle {
+    font-size: 16px;
+    line-height: 16px;
+    margin-bottom: 3.5vw;
+  }
+
+  .order__description {
+    font-size: 15px;
+    line-height: 21px;
+    margin-bottom: 5vw;
+  }
+
+  .color-list {
+    height: 38px;
+    margin-bottom: 5vw;
+    margin-right: 10vw;
+  }
+
+  .list__item {
+    margin-right: 1.4vw;
+  }
+
+  .list__item_grey,
+  .list__item_blue,
+  .list__item_green {
+    width: 33.18px;
+    height: 33px;
+  }
+
+  .list__item_pink {
+    width: 38px;
+    height: 38px;
+  }
+
+  .order__price {
+    font-size: 38px;
+    line-height: 52px;
+  }
+
+  .order-number {
+    margin-left: 1vw;
+    margin-right: 5vw;
+    padding-left: 2.1vw;
+    padding-top: 14px; 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5841754)